src/encoded/static/components/genome_browser.js

```diff
@@ -16,6 +16,145 @@ export const annotationTypeMap = {
     'representative DNase hypersensitivity sites (rDHSs)': 'rDHS',
 };
 
+const GV_COORDINATES_KEY = 'ENCODE-GV-coordinates';
+
+/**
+ * Returns Valis coordinates off the bar user inputs data in
+ *
+ * @param {string} scrollLocation, in format: "chr{contig} {x0}bp to {x1}bp, where contig, x0 and x2  and numbersp"
```

Comment:
"are numbers"?

Reply:
Also, I think it can also be in `Mbp`. I was actually wondering if it could be in anything other than bp and Mbp - do you know?